eralchemy2/models.py

```diff
@@ -117,8 +123,8 @@ class Relation(Drawable):
         "+": "1..n",
     }
 
-    @staticmethod
-    def make_from_match(match):
+    @classmethod
+    def make_from_match(cls, match: re.Match) -> Relation:
```

Comment:
why make this a classmethod if there is no class state which is referenced or changed?